http/objectstore/store/store.go

```diff
@@ -0,0 +1,117 @@
+package store
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/filedag-project/filedag-storage/http/objectstore/uleveldb"
+	"io"
+	"io/ioutil"
+	"strings"
+	"time"
+)
+
+//StorageSys store sys
+type StorageSys struct {
+	db      *uleveldb.ULevelDB
+	dagPool dagPoolClient
+}
+
+const objectPrefixTemplate = "object-%s-%s-%s/"
+const allObjectPrefixTemplate = "object-%s-%s-"
+
+//StoreObject store object
+func (s *StorageSys) StoreObject(user, bucket, object string, reader io.Reader) (ObjectInfo, error) {
+	if strings.HasPrefix(object, "/") {
+		object = object[1:]
+	}
+	cid, err := s.dagPool.PutFile(bucket, object, reader)
+	if err != nil {
+		return ObjectInfo{}, err
+	}
+	all, err := ioutil.ReadAll(reader)
+	meta := ObjectInfo{
+		Bucket:                     bucket,
+		Name:                       object,
+		ModTime:                    time.Now().UTC(),
+		Size:                       int64(len(all)),
+		IsDir:                      false,
+		ETag:                       cid,
+		VersionID:                  "",
+		IsLatest:                   false,
+		DeleteMarker:               false,
+		RestoreExpires:             time.Unix(0, 0).UTC(),
+		RestoreOngoing:             false,
+		ContentType:                "application/x-msdownload",
+		ContentEncoding:            "",
+		Expires:                    time.Unix(0, 0).UTC(),
+		StorageClass:               "STANDARD",
+		UserDefined:                nil,
+		UserTags:                   "",
+		Parts:                      nil,
+		Writer:                     nil,
+		Reader:                     nil,
+		PutObjReader:               nil,
+		AccTime:                    time.Unix(0, 0).UTC(),
+		Legacy:                     false,
+		VersionPurgeStatusInternal: "",
+		VersionPurgeStatus:         "",
+		NumVersions:                0,
+		SuccessorModTime:           time.Now().UTC(),
+	}
```

Comment:
Need a reconsideration about how much the meta info fields we want to save to db. It seems too many.

Reply:
Okay, I'll check it out and re-determine what data we're going to store later